scout/server/links.py

```diff
@@ -593,7 +594,11 @@ def gnomad_sv_link(variant_obj, build=37):
     else:
         url_template += f"-{variant_obj['position']}"
 
-    url_template += "?dataset=gnomad_sv_r2_1"
+    if build == 37 or variant_obj["chromosome"] in ["M", "MT"]:
+        url_template += "?dataset=gnomad_sv_r2_1"
+        return url_template
+
+    url_template += "?dataset=gnomad_sv_r4"
```

Comment:
So just for my own curiosity: SVs in v4 are using hg38, but the MT ones still need to use the old set?

Reply:
Yes, it's a bit of a version mess on their side. Several features are still only available in hg37, like gene mutation intolerance scores. They have a whole table of what version to use for what feature. 🙄 Management by research release cycle - they work really hard towards delivering something cool for the ASHG meeting each year.

Reply:
I guess it is nice to use hg38 for some SVs at least!